### PR TITLE
Add AI Transcription Guide

### PR DESCRIPTION
**Pull request by giraffekey**
# AI Transcription Guide

I wrote this assuming that https://github.com/nkkko/sapat/pull/1 will be accepted.

/claim #13

## Writer's Checklist

- [x] **Follow Writing Structure**
  - [x] I have used more than two H2 subtitles to organize content.
  - [x] Include an introduction with at least two paragraphs before the first H2 section.
  - [x] Use bold, italic, underline, notes, quotes, TLDRs, and key points as needed.
  - [x] Include `code elements` where appropriate.
  - [x] Write the article in Markdown format.

- [x] **Fact-Check**
  - [x] I have verified all facts and data points included in the article.

- [ ] **Assets**
  - [ ] Save images in the `/assets` folder.
  - [ ] Follow naming conventions: `YYYYMMDD_title_of_the_article_IMG_NAME_NO.png`.

- [x] **Interlinking**
  - [x] Use `CTRL+F` to search for relevant keywords on:
    - [Daytona Dotfiles Sitemap](https://www.daytona.io/sitemap-dotfiles.xml)
    - [Daytona Definitions Sitemap](https://www.daytona.io/sitemap-definitions.xml)
  - [x] Interlink keywords using the appropriate URLs.

- [x] **Glossary/Definitions**
  - [x] Identify and mark new terms that could be added to the Definitions page.

- [x] **Review and Edit**
  - [x] Read the article out loud to catch any awkward phrasing.
  - [x] Run the draft through [Grammarly](https://grammarly.com) or a similar tool to correct grammar and spelling errors.
  - [x] Double-check adherence to the style guide and repository guidelines.

### CODE REVIEW

File: guides/20240819_ai_transcription.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
+---
+title: "Transcribe Video and Audio with Python"
+description: "This guide provides a comprehensive step-by-step process for setting up and using Sapat to automatically transcribe video and audio files using Azure OpenAI's Whisper API."
```
Comment (nkkko) [inline guides/20240819_ai_transcription.md:3]: Maybe to be more generic and just state OpenaAI Whisper compatible API or whatever is correct?

File: guides/20240819_ai_transcription.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+By following this guide, you will learn how to set up the necessary environment, install the required software, and use the [transcription](/definitions/transcription.md) tool. The tool supports processing individual video files or entire directories, ensuring temporary MP3 files are cleaned up after [transcription](/definitions/transcription.md). You can choose between using Azure OpenAI's Whisper API or Rev AI API based on your preference and access. A basic understanding of Python and command-line operations, as well as access to the relevant API, is required.
+
+### Prerequisites
+
```
Comment (nkkko) [inline guides/20240819_ai_transcription.md:17]: only prereq should be daytona and repo with appropriate devcontainer that will set all of the prereq which I have in the repo. 

with every article the main idea is to show how daytona simplifies development, quick hacking/prototyping, oss contributions, etc.

**Review by nkkko — CHANGES_REQUESTED** [review]
Overall 5/10, could you rewrite it a bit to focus on motivating developers to play with ai tools like whisper, get them introduced to api, show them how they can quickly prototype such project with Daytona. The idea is to show the magic of quickly spinning the project and getting your ideas into functioning code.

Could you please add some diagrams or images to illustrate it a bit.

Thank you.

**Comment by nkkko** [comment]
Please be mindful of the checklist and don't check stuff if it is not done, for example assets. Thanks.

**Comment by mojafa** [comment]
@giraffekey guide fails DCO check

**Comment by mojafa** [comment]
@giraffekey sorry for the delay. Please manually close all the other PRs you have open. I'll assign the issue this PR is tackling to you. You can always submit any other PR if the issues are open still, as we require one PR per contributor.

**Comment by mojafa** [comment]
I'm reviwing this PR now.

**Pull request closed without merging**